ts/src/woo.ts

```diff
@@ -825,10 +880,12 @@ export default class woo extends Exchange {
          * @param {float} amount how much of currency you want to trade in units of base currency
          * @param {float|undefined} price the price at which the order is to be fullfilled, in units of the quote currency, ignored in market orders
          * @param {object} params extra parameters specific to the woo api endpoint
+         * @param {bool} params.stop True if is a algo order
+         * @param {string} params.triggerPrice algo order's triggerPrice stopLossPrice = takeProfitPrice = stopPrice = triggerPrice
          * @returns {object} an [order structure]{@link https://docs.ccxt.com/#/?id=order-structure}
          */
         await this.loadMarkets ();
-        const market = this.market (symbol);
+        // const market = this.market (symbol);
```

Comment:
@sc0Vu why is this commented?

Reply:
oh, it's not necessarily to use market params. I'll remove it.